ckanext/security/mailer.py

```diff
@@ -3,10 +3,12 @@
 import codecs
 import logging
 import six
+import flask
```

Comment:
We shouldn't import Flask before we know which CKAN version we have; it may not be available.

Reply:
Yep, will fix :+1: